code/datums/mutations/speech.dm

```diff
@@ -230,3 +230,55 @@
 	..()
 	owner.grant_language(/datum/language/common)
 	owner.remove_language(/datum/language/beachbum)
+
+
+	/datum/mutation/human/italian
+	name = "Italian"
+	quality = MINOR_NEGATIVE
+	dna_block = NON_SCANNABLE
+	text_gain_indication = "<span class='notice'>You feel-a the spicy meatballs and-a pasta singing to your-a soul!</span>"
+	text_lose_indication = "<span class='notice'>You no longer feel the urge to switch sides midway through a war. Your love of pasta passes.</span>"
+
+/datum/mutation/human/italian/say_mod(message)
+	if(message)
+		message = " [message] "
+		message = replacetext(message,"mom","mamma")
+		message = replacetext(message,"dad","pappa")
+		message = replacetext(message,"baby",pick("bambino","little sausage roll"))
+		message = replacetext(message,"spicy","a-spicy")
+		message = replacetext(message,"I'm","I'm-a")
+		message = replacetext(message,"friend","enemy")
+		message = replacetext(message,"enemy","friend")
```

Comment:
why?

Reply:
Two world wars

Reply:
Not very funny, but could be annoying in game.

Reply:
This is going to result in enemy being replaced by friend, and friend staying in tact, since you directly replace all friend with enemy, then after that enemy with friend, which includes all enemy and doesn't distinguish with enemy that used to be friend.

Reply:
you should be able to just put an italian spin both words to prevent it

Reply:
I'll add the -a then

Reply:
ene-mia

Reply:
Actually, the triple alliance was defensive and the Austrians attacked first

Reply:
paesano

Reply:
@grazyn go home nazi